engines/instance_verification/spec/requests/api/connect/v3/systems/products_controller_spec.rb

```diff
@@ -110,10 +113,14 @@
         before do
           expect(InstanceVerification::Providers::Example).to receive(:new)
             .with(be_a(ActiveSupport::Logger), be_a(ActionDispatch::Request), payload_sap, instance_data).and_call_original
-
-          expect(Rails.cache).to receive(:write).with(
-            ['127.0.0.1', system.login, product_sap.id].join('-'), true, expires_in: 20.minutes
+          allow(InstanceVerification).to receive(:cache_config).and_return(
+            'REPOSITORY_CLIENT_CACHE_DIRECTORY' => 'cache_repo_dir',
+            'REGISTRY_CLIENT_CACHE_DIRECTORY' => 'cache_registry_dir'
```

Comment:
This should be `REPOSITORY_CACHE_EXPIRY_MINUTES` and have hardcoded 20, right?

Reply:
The value is a harcoded `20` yes but no , that line should be replaced by `REPOSITORY_CACHE_EXPIRY_MINUTES`, should be cleanup from the test
we write the cache to the directory so we need `_DIRECTORY` variable

Reply:
But for this PR `REGISTRY_CLIENT_CACHE_DIRECTORY` does not exist???